ts/types/MessageWithContentAndDueDatePO.ts

```diff
@@ -0,0 +1,15 @@
+import { MessageContent } from "../../definitions/backend/MessageContent";
+import { MessageWithContentPO } from "./MessageWithContentPO";
+
+type MessageContentWithDueDate = MessageContent & {
```

Comment:
you can use a utility method called `requireProperty` that we used for pagopa - see an example here https://github.com/teamdigitale/italia-app/blob/master/ts/types/pagopa.ts#L52